scripts/README.md

```diff
@@ -36,3 +36,44 @@ python3 -m src.plugin_runner.load_submissions_graph /path/to/input/file /path/to
 ```text
 pip3 install -r requirements.txt
 ```
+
+# Clustering code submissions
```

Comment:
Also, could you please add more information into README what is the difference between all tasks, currently, it is not obvious, especially for external users. Probably, it is better to describe the whole pipeline here: wich tasks and in wich order should be runned